cpp/turbodbc/Library/src/field_translators/int64_translator.cpp

```diff
@@ -11,7 +11,7 @@ int64_translator::~int64_translator() = default;
 
 field int64_translator::do_make_field(char const * data_pointer) const
 {
-	return {*reinterpret_cast<long const *>(data_pointer)};
+	return {*reinterpret_cast<intptr_t const *>(data_pointer)};
```

Comment:
This will translate to a 64 bit integer by design because the binding uses `SQL_BIGINT`. Please replace this with `int64_t`